src/app/controllers/notifications_controller.rb

```diff
@@ -0,0 +1,15 @@
+class NotificationsController < ApplicationController
+  LIMIT = 30
+
+  def index
```

Comment:
This controller must be protected, as it will raise an exception if the user is a guest